### ISSUE TITLE
zpk2tf, tf2zpk representation

### ISSUE THREAD

**Issue by xkzl** [kfrlib/kfr#191]
Hello,

I would be interested in using tf representation for a filter that I designed. 

Is there any existing interface to turn tf coefficients into zpk representation and opposite ?

It seems I am not the only one in need. https://github.com/kfrlib/kfr/issues/148

I have found a zpk2tf method but it is marked as internal. This tf2zpk would be great feature to implement.
Could you advice on this ? @dancazarin 

Thank you

**Comment by dancazarin** [comment]
Hello,

If an external solution is appropriate, python's scipy.signal.tf2zpk is compatible by all means with KFR and can be used to transform from tf to zpk representation. Or do you need a builtin function to do this at runtime?

Internal functions are absolutely ok for using in your code but may be changed/removed in future versions without any prior notice (not part of stable API).

**Comment by xkzl** [comment]
yes I definitely don't know whether kfr is able to solve polynomials, but this was the main limitation in my case.
